src/main/java/io/github/moulberry/notenoughupdates/listener/NEUEventListener.java

```diff
@@ -149,6 +146,17 @@ public void onTick(TickEvent.ClientTickEvent event) {
 			}
 		}
 
+		//MiningOverlay and TimersOverlay need real tick speed
+		if (neu.hasSkyblockScoreboard()) {
```

Comment:
NEUs slow (and centralized) ticking is starting to bother me in places. Perhaps in the future we could work on an overall more reactive coding style. (This seems like a fine bodge tho)